github-for-jira.sd.yml

```diff
@@ -149,6 +149,16 @@ resources:
         # Name of the RDS resource from above
         resource: rds
 
+  - name: deployment-history-cache
+    type: dynamo-db
+    attributes: &table-attributes
+      HashKeyName: Id
+      HashKeyType: "S"
+      RangeKeyName: StatusCreatedAt
```

Comment:
Should we also change that to be generic?

Reply:
Changed.